app/domain/logs.rb

```diff
@@ -489,14 +490,69 @@ module AuthnJwt
         code: "CONJ00096D"
       )
 
-      FOUND_JWT_FIELD_IN_TOKEN = ::Util::TrackableLogMessageClass.new(
+      FoundJwtFieldInToken = ::Util::TrackableLogMessageClass.new(
         msg: "Successfully found field '{0}' in token and its value is '{1}'",
         code: "CONJ00097D"
       )
 
-      FOUND_JWT_IDENTITY= ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully found jwt identity '{0}'",
-        code: "CONJ00098D"
+      FoundJwtIdentity= ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully found JWT identity '{0}'",
+        code: "CONJ00098I"
+      )
+
+      ValidatedJwtStatusConfiguration = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated JWT status configuration",
+        code: "CONJ00099I"
+      )
+
+      ValidatedAccountExists = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated that account exists",
+        code: "CONJ00100D"
+      )
+
+      ValidateAndDecodeTokenPassed = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully passed validate and decode token",
+        code: "CONJ00101D"
+      )
+
+      GetJwtIdentityPassed = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully passed get JWT identity",
+        code: "CONJ00102D"
+      )
+
+      ValidateRestrictionsPassed = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully passed JWT validate restrictions",
```

Comment:
```suggestion
        msg: "Successfully validated JWT restrictions",
 ```
@nessiLahav  - let's discuss this

Reply:
fixed as we agreed please review